app/main.py

```diff
@@ -1 +1,12 @@
-# write your code here
+def calculate_team_total_rating(team_list: list) -> int:
+    return sum([player.get_rating() for player in team_list])
```

Comment:
```suggestion
    return sum(player.get_rating() for player in team_list)
```
no need to create list